README.md

```diff
@@ -4,20 +4,20 @@ ESLint and Prettier config
 
 ## Integration
 
-- Install the package along with it’s peer dependencies:
+- First, install the package along with it’s peer dependencies:
```

Comment:
, I have a feeling people might be only half–reading these. Maybe if we structure it like this?:

- First, install the package along with it’s peer dependencies:

  ```
  (…)
  ```

- Then:

  - if your project uses TypeScript:

    ```
    (…)
    ```

  - if your project _does not_ TypeScript, but uses Babel:

    ```
    (…)
    ```